### ISSUE TITLE
Reverse X Axis but still have zero at the bottom

### ISSUE THREAD

**Issue by batesy87** [google/google-visualization-issues#2389]
Wondering if there's a way to produce a column chart with the origin 0 on the Y Axis but the scale in reverse order with 1 at the top and the largest number at the bottom. I want to show google rankings for search terms and have a rank of 1 appear as a bigger bar than a rank of 100 but still grow from the bottom.

Example
![example](https://cloud.githubusercontent.com/assets/1161433/20184083/5e314fdc-a75f-11e6-8869-7e639520fea9.png)

**Comment by dlaliberte** [comment]
Yes. Use direction: -1 on the vAxis, and baseline: 100 (or whatever your highest value is).  If you really want the tick values to end in 1, you'll probably have to specify all the tick values yourself using the ticks option.
